checks.d/tokumx.py

```diff
@@ -424,7 +424,11 @@ def collect_metrics(self, instance, server, conn, db, tags):
                 db_tags = list(tags)
                 db_tags.append('db:%s' % dbname)
                 db = conn[dbname]
-                stats = db.command('dbstats')
+                try:
+                    stats = db.command('dbstats')
+                except:
```

Comment:
Thanks, totally agree. Here's the error message without this `except`:

```
OperationFailure: command SON([('dbstats', 1)]) on namespace *.$cmd failed: unauthorized
```

I assume this means `except OperationFailure:` is what I want?